### DISCUSSION TITLE
Which is the most effective way to modify/enrich a solution deployed from Azure AI Studio?

### DISCUSSION THREAD

**Discussion by davidcervigon** [microsoft/sample-app-aoai-chatGPT#473] [Q&A]
Hi

Looks like Azure AI Studio just deploys this repo and seed the environment variables in the Azure App Service with the endpoints, connections, components, indexes etc. modeled for the project. How can a deployed solution be customized or modified (for instance, to embed the system message in the code instead of using Environment variables - to overcome the 50000 chars limitation), beyond connecting to the App Service using SSH?. Or, how can this repo be deployed using the same configuration modeled in the Studio?

Thanks

**Comment by ntabernacle** [comment]
Best way I've found so far is to use prompt flow in AI Studio then use this app to point to your custom deployed end point. The system message will then be run from the deployed prompt flow.

**Reply by JukkaAGit** [reply]
How do you actually do this? I'm able to create prompt flow and access it with postman for example, but where I need to put the endpoint in sample-app-aoai-chatGPT?
